source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -32,315 +32,136 @@ The procedure on this page documents the following:
-              - targets: [minio.example.net]
-
-
-   .. tab-item:: Nodes
-
-      Node metrics are specific for node-level monitoring. You need to list all MinIO nodes for this configuration.
-
-      .. code-block:: yaml
-         :class: copyable
-      
-         global:
-            scrape_interval: 60s
-         
-         scrape_configs:
-            - job_name: minio-job-node
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/node
-              scheme: https
-              static_configs:
-              - targets: [minio-1.example.net, minio-2.example.net, minio-N.example.net]
-
-	      
-   .. tab-item:: Bucket
-
-      .. code-block:: yaml
-         :class: copyable
-      
-         global:
-            scrape_interval: 60s
-         
-         scrape_configs:
-            - job_name: minio-job-bucket
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/bucket
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-
-   .. tab-item:: Resource
-
-      .. code-block:: yaml
-         :class: copyable
-
-         global:
-            scrape_interval: 60s
-
-         scrape_configs:
-            - job_name: minio-job-resource
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/resource
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-
-Start the Prometheus cluster using the configuration file:
+.. code-block:: yaml
+   :class: copyable
 
-.. code-block:: shell
+   scrape_configs:
+   - job_name: minio-job
+     bearer_token: TOKEN
+     metrics_path: /minio/metrics/v3
+     scheme: https
+     static_configs:
+     - targets: [minio.example.net]
+
+To scrape multiple types of metrics, run :mc-cmd:`mc admin prometheus generate --api-version v3 <mc admin prometheus generate --api-version>` for each type and add the ``job_name`` section to the ``scrape_configs`` in your Prometheus configuration.
+
+The following example scrapes audit and system metrics every 60 seconds:
+
+.. code-block:: yaml
    :class: copyable
 
-   prometheus --config.file=prometheus.yaml
+   global:
+     scrape_interval: 60s
 
-3) Analyze Collected Metrics
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   scrape_configs:
+   - job_name: minio-job-audit
+     bearer_token: TOKEN
+     metrics_path: /minio/metrics/v3/audit
+     scheme: https
+     static_configs:
+     - targets: [minio.example.net]
 
-Prometheus includes an :prometheus-docs:`expression browser <prometheus/latest/getting_started/#using-the-expression-browser>`. 
-You can execute queries here to analyze the collected metrics.
+   - job_name: minio-job-system
+     bearer_token: TOKEN
+     metrics_path: /minio/metrics/v3/system
+     scheme: https
+     static_configs:
+     - targets: [minio.example.net]
 
-.. tab-set::
+If needed, edit the generated configuration for your environment.
+Common changes include:
```

Comment:
There's a lot of stuff here for an unordered list. But tables suck. Ideas?

Reply:
Since we aren't trying to be exhaustive with covering all the possible changes they might make, tabs might work here.